person.rb

```diff
@@ -1,14 +1,15 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :age, :name
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
```

Comment:
- [ ] We are supposed to have a method that adds a rental, something like `add_rental`

Reply:
Hi @descholar-ceo, thanks for your feedback, I do that.